code/modules/reagents/machinery/mortar.dm

```diff
@@ -70,9 +24,11 @@
 	if (crushable)
 		to_chat(user, "<span class ='warning'>There's already something inside!</span>")
 		return 1
-	if (!is_type_in_list(O, blend_items) && !is_type_in_list(O, juice_items))
-		to_chat(user, "<span class ='warning'>You can't grind that!</span>")
-		return ..()
+
+	//There used to be a check here to only allow certain whitelisted things to be put inside the mortar. Now we just check size. God help us.
```

Comment:
There is no whitelist because that's hard coding. Literally the entire purpose of the PR is to eliminate the whitelist. This was *your idea for implementation 18 months ago*.